.idea/misc.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.9 (allotropy)" project-jdk-type="Python SDK" />
+<component name="ProjectRootManager" version="2" project-jdk-name="Python 3.9 (allotropy)" project-jdk-type="Python SDK" />
```

Comment:
```suggestion
  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.9 (allotropy)" project-jdk-type="Python SDK" />
```

I can't commit this for you, but this should eliminate the changes in this file.